### PR TITLE
Exposes several parameters from the somatic LLR filter

### PR DESCRIPTION
**Pull request by chrisamiller**
This enables tuning the pipeline to be appropriate for low-purity tumors

The somatic pipeline test runs successfully, with or without these params specified.

### CODE REVIEW

File: definitions/pipelines/somatic_exome.cwl [hunk 1]
```diff
@@ -562,6 +577,7 @@ steps:
             normal_sample_name: normal_sample_name
             vep_custom_annotations: vep_custom_annotations
             known_variants: known_variants
+
```
Comment (tmooney) [inline definitions/pipelines/somatic_exome.cwl:580]: I wouldn't separate the `out` from the rest of the step definition unless we're going to pursue some large-scale formatting changes.

**Review by tmooney — DISMISSED** [review]
+1 🦙

**Comment by chrisamiller** [comment]
Update with whitespace fix

**Review by tmooney — APPROVED** [review]
+1 🧇

**Pull request merged**